### PR TITLE
Multicluster: kind support

### PR DESCRIPTION
**Pull request by froesef**
This PR Adds support to spin up two kind clusters in the same dev environment. 
These kind clusters share one registry so that images only have to be uploaded once.

### CODE REVIEW

File: website/content/contribute/code/flow/index.md [hunk 1]
```diff
@@ -120,6 +120,17 @@ Running end to end tests:
 make e2e
 ```
 
+# Building in a multi cluster environment
+
+As the-mesh-for-data is can run in a multi-cluster environment there is also a test environment
```
Comment (roee88) [inline website/content/contribute/code/flow/index.md:125]:
```suggestion
As {{< name >}} can run in a multi-cluster environment there is also a test environment
```

File: hack/tools/create_kind.sh [hunk 2]
```diff
@@ -63,14 +63,33 @@ kind_delete() {
...
         bin/kubectl config use-context kind-kind
-        for node in $(kind get nodes); do
+        for node in $(kind get nodes --name kind); do
+          bin/kubectl annotate node "${node}" "tilt.dev/registry=kind-registry:5000";
+          docker cp ../registry/ca.crt "$node":/usr/local/share/ca-certificates
+          docker exec "$node" update-ca-certificates
+        done
+}
+
+kind_create_control() {
```
Comment (roee88) [inline hack/tools/create_kind.sh:85]: why not create a function that accepts name and yaml location

File: hack/tools/create_kind.sh [hunk 3]
```diff
@@ -83,6 +102,15 @@ case "$op" in
         registry_delete || true
         certs_delete || true
         kind_delete || true
+        kind_delete_control || true
+        ;;
+    multi)
```
Comment (roee88) [inline hack/tools/create_kind.sh:107]: I wonder if we would want to create/delete one cluster independently

File: hack/tools/kind-control-config.yaml [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+  extraPortMappings:
+  - containerPort: 30333  # Razee dash API node port in K8s
+    hostPort: 3333  # Razee dash API on host
+  - containerPort: 30080  # Razee dash UI node port in K8s
+    hostPort: 8080  # Razee dash UI on host
```
Comment (roee88) [inline hack/tools/kind-control-config.yaml:12]: Eventually we may not want to expose the UI to the outsiders but rather just port-forward to access it.

**Review by roee88 — COMMENTED** [review]
I wonder if

File: hack/tools/kind-control-config.yaml (hunk 4 above)
Comment (froesef) [reply hack/tools/kind-control-config.yaml:12]: Yes but I think for local development in kind it's easier to have this exposed right now.

File: hack/tools/create_kind.sh [hunk 5]
```diff
@@ -63,14 +63,33 @@ kind_delete() {
         bin/kind delete cluster --name kind
 }
 
+kind_delete_control() {
+        bin/kind version
```
[the remark this answers is not in the document]
Comment (froesef) [reply hack/tools/create_kind.sh]: Removed

File: hack/tools/create_kind.sh (hunk 3 above)
Comment (froesef) [reply hack/tools/create_kind.sh:86]: I used parameters for create and delete now. Does that also solve this comment?

Comment (roee88) [reply hack/tools/create_kind.sh:86]: See https://github.com/IBM/the-mesh-for-data/pull/142#pullrequestreview-534300178 to understand what I meant, but we are good for now.

File: website/content/contribute/code/flow/index.md [hunk 6]
```diff
@@ -120,6 +120,17 @@ Running end to end tests:
 make e2e
 ```
 
+# Building in a multi cluster environment
+
+As {{< name >}} is can run in a multi-cluster environment there is also a test environment
```
Comment (roee88) [inline website/content/contribute/code/flow/index.md:125]: Why "is can"?

**Review by roee88 — APPROVED** [review]

**Pull request merged**